lib/lookup.js

```diff
@@ -1,7 +1,8 @@
 'use strict';
-const normgit = require('normalize-git-url');
-const util = require('util');
 const path = require('path');
+const util = require('util');
+
+const normgit = require('normalize-git-url');
```

Comment:
I kept the style of the project, that is to separate builtin, npm and local requires.